src/Resources/ResourceManager/Implementation/Policy/PolicyHelpStrings.cs

```diff
@@ -124,5 +124,39 @@ public static class PolicyHelpStrings
+        public const string NewPolicyExemptionDisplayNameHelp = "The display name for the new policy exemption.";
+        public const string NewPolicyExemptionDescriptionHelp = "The description for the new policy exemption.";
+        public const string NewPolicyExemptionExpiresOnHelp = "The expiration date and time (in UTC ISO 8601 format yyyy-MM-ddTHH:mm:ssZ) of the new policy exemption.";
+        public const string NewPolicyExemptionPolicyAssignmentIdHelp = "The referenced policy assignment Id for the new policy exemption.";
+        public const string NewPolicyExemptionPolicyDefinitionReferenceIdsHelp = "The policy definition reference ID list when the associated policy assignment is an assignment of a policy set definition.";
+        public const string NewPolicyExemptionMetadataHelp = "The metadata for the new policy exemption. This can either be a path to a file containing the metadata JSON, or the metadata as a JSON string.";
+        public const string RemovePolicyExemptionNameHelp = "The name of the policy exemption to delete.";
+        public const string RemovePolicyExemptionScopeHelp = "The scope of the policy exemption to delete, e.g. /providers/managementGroups/{managementGroupName}, defaults to current subscription.";
+        public const string RemovePolicyExemptionIdHelp = "The fully qualified policy exemption ID to delete, including the scope, e.g. /subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.Authorization/policyExemptions/{policyExemptionName}.";
+        public const string RemovePolicyExemptionInputObjectHelp = "The policy exemption object to remove that was output from another cmdlet.";
+        public const string SetPolicyExemptionNameHelp = "The name of the policy exemption to update.";
+        public const string SetPolicyExemptionScopeHelp = "The scope of the updated policy exemption, e.g. /providers/managementGroups/{managementGroupName}, defaults to current subscription.";
+        public const string SetPolicyExemptionIdHelp = "The fully qualified policy exemption Id to update, including the scope, e.g. /subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.Authorization/policyExemptions/{policyExemptionName}.";
+        public const string SetPolicyExemptionCategoryHelp = "The policy exemption category of the updated policy exemption. Possible values are Waiver and Mitigated.";
+        public const string SetPolicyExemptionDisplayNameHelp = "The display name for the updated policy exemption.";
+        public const string SetPolicyExemptionDescriptionHelp = "The description for the updated policy exemption.";
+        public const string SetPolicyExemptionExpiresOnHelp = "The expiration date and time (in UTC ISO 8601 format yyyy-MM-ddTHH:mm:ssZ) of the updated policy exemption.";
+        public const string SetPolicyExemptionClearExpirationHelp = "whether clear the expiration date and time of the updated policy exemption";
```

Comment:
```suggestion
        public const string SetPolicyExemptionClearExpirationHelp = "Whether to clear the expiration date and time of the updated policy exemption.";
```

I took a guess, I'm not really clear on what this is for.

Reply:
When the exemption was set a expiration datetime. The user can use this parameter to clear the expiration date on the existing exemption. What do you suggest to change the statement to?

Reply:
Updated my suggestion based on that.